tests/test_kubernetes_tools.py

```diff
@@ -4486,7 +4486,7 @@ def test_warning_big_bounce_routable_pod():
             job_config.format_kubernetes_app().spec.template.metadata.labels[
                 "paasta.yelp.com/config_sha"
             ]
-            == "config45672f04"
+            == "config4ed19ece"
```

Comment:
I configured adding a conditional to `get_termination_grace_period` to have it return None if there isn't an explicit override of `pre_stop_drain_seconds` in yelpsoa-configs, but decided I'd rather have a big bounce now than additional code complexity forever.